packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -719,7 +776,7 @@ class FlameChartNode<T> {
 
   static const _selectedNodeColor = lightSelection;
   static const _alternateTextColor = Colors.black;
-  static const _minWidthForText = 22.0;
+  static const _minWidthForText = 30.0;
```

Comment:
why was this edited as part of this CL? performance concerns?
if so maybe note that we would prefer for this to be smaller but performance on zoom doesn't allow it.